Server/User/routers/notifications_internal.py

```
@@ -0,0 +1,115 @@
+
+# Internal endpoints for service-to-service communication
+@router.get("/internal")
+async def get_notifications_internal(
+    x_user_id: str = Header(alias="X-User-ID"),
+    x_service_token: str = Header(alias="X-Service-Token"),
+    db: Session = Depends(get_db)
+):
+    """Internal endpoint for getting notifications (for Psychologist service)"""
+    try:
+        # Validate service token
+        if x_service_token != "faceofmind_internal_service_2024":
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "x_service_token"

(S105)